include/ibamr/SpongeLayerForceFunction.h

```diff
@@ -146,6 +146,7 @@ class SpongeLayerForceFunction : public IBTK::CartGridFunction
                             double kappa,
                             SAMRAI::tbox::Pointer<SAMRAI::hier::Patch<NDIM> > patch);
 
+    double d_chi;
```

Comment:
Could we add a documentation string for this function?

Also, we usually set a default value in the header. I think it would be reasonable to set it to `quiet_NaN`.